package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.0.0",
   "private": true,
   "description": "TypeScript template action",
-  "main": "lib/main.js",
+  "main": "src/main.ts",
```

Comment:
Should this be dist/index.js since that's what gets executed?

Reply:
What gets executed is defined in `action.yml`

```yaml
runs:
  using: 'node16'
  main: 'dist/index.js'
```

Reply:
src/main.ts is what `ncc` uses to bundle the dist folder